chain/jsonrpc-primitives/src/types/blocks.rs

```diff
@@ -61,13 +62,13 @@ impl From<near_client_primitives::types::GetBlockError> for RpcBlockError {
             near_client_primitives::types::GetBlockError::IOError { error_message } => {
                 Self::InternalError { error_message }
             }
-            near_client_primitives::types::GetBlockError::Unreachable { error_message } => {
+            near_client_primitives::types::GetBlockError::Unreachable { ref error_message } => {
```

Comment:
Why? I mean, it won't hurt, but I think we don't need it, so I wonder about the reason to be explicit here

Reply:
"value partially moved here"

Reply:
I see! All good